agent/config/runtime_test.go

```diff
@@ -5473,7 +5473,7 @@ func TestLoad_FullConfig(t *testing.T) {
 		HTTPSPort:                              15127,
 		HTTPUseCache:                           false,
 		KeyFile:                                "IEkkwgIA",
-		KVMaxValueSize:                         1234567800000000,
+		KVMaxValueSize:                         1234567800,
```

Comment:
Good question! I left some detail in the commit message for f34d3543b11aa1a42ea0880df6dfb533b3ce541e

The hcl decoding apparently uses `strconv.ParseInt`, which fails to parse a 64bit int on a 32bit platform. Since hcl v1 is basically EOL, it seems unlikely we'll fix this in hcl. We run these tests with both json and hcl input, and only the hcl input was failing.

Since this test is only about loading values from config files, the extra large number doesn't seem important, so I trimmed the number a bit to fit into 32bit.